pykafka/simpleconsumer.py

```diff
@@ -194,7 +194,7 @@ def start(self):
 
         # Figure out which offset wer're starting on
         if self._reset_offset_on_start:
-            self._reset_offsets()
+            self.reset_offsets()
             # make sure the reset is saved in kafka before it can rebalance
```

Comment:
From your first sentence, it sounds more like you mean something like this:
```python
if self._reset_offset_on_start:
    self.reset_offsets()
def reset_offsets(self, ...):
    ...
    self.commit_offsets()
```
This makes a lot of sense to me, but it doesn't match the code you provided. I believe it solves the error case you mention. Am I missing something?